front-end/src/views/GeneratePosterPage/index.vue

```diff
@@ -96,9 +110,20 @@
     data() {
       return {
         form: {},
+        communes: []
       };
     },
     methods: {
+      async search() {
+        if(!this.form.commune) {
+          this.communes = [];
+          return;
+        }
+        const queryUrl = "https://api-adresse.data.gouv.fr/search/?q="+this.form.commune+"&type=municipality&autocomplete=1";
+        const response = await fetch(queryUrl);
+        const { features: communes } = await response.json();
+        this.communes = communes;
```

Comment:
Je trouve ça pas super lisible comme écriture, qu'est ce que tu penses de :
`this.communes = await response.json().features;` ?